test/AutoDiff/control_flow_sil.swift

```diff
@@ -1,9 +1,11 @@
 // RUN: %target-swift-frontend -emit-sil -verify %s | %FileCheck %s -check-prefix=CHECK-DATA-STRUCTURES
-// RUN: %target-swift-frontend -emit-sil -verify %s | %FileCheck %s -check-prefix=CHECK-SIL
+// RUN: %target-swift-frontend -emit-sil -verify -Xllvm -sil-print-after=differentiation -o /dev/null 2>&1 %s | %FileCheck %s -check-prefix=CHECK-SIL
```

Comment:
Note: checking `-Xllvm -sil-print-after=differentiation` here is nicer because:
* The printed SIL matches the SIL printed by `-Xllvm -debug-only=differentiation`.
* `-emit-sil` performs further optimizations (e.g. mandatory inlining), so SIL contains floating-point builtins, etc.